openstack/vpn/v5/customer_gateways/request.go

```diff
@@ -0,0 +1,26 @@
+package customer_gateways
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+func ListAll(c *golangsdk.ServiceClient, opts *ListOpts) ([]CustomerGateway, error) {
```

Comment:
if the function named **ListAll**, why we need ListOpts as a parameter?

Reply:
removed